packages/core-v2/src/semantic-model/aggregator/aggregator.ts

```diff
@@ -63,15 +77,16 @@ class SemanticModelAggregatorInternal implements SemanticModelAggregator {
         this.models.set(model, unsubscribe);
 
         callback(model.getEntities(), []);
-    };
+    }
 
     deleteModel(model: SupportedModels) {
         const unsubscribe = this.models.get(model);
         if (!unsubscribe) {
-            throw new Error('Model not added.');
+            throw new Error("Model not added.");
         }
         this.models.delete(model);
         this.notifyBaseModel(model, [], Object.keys(model.getEntities()));
+        // TODO: shouldn't `unsubscribe()` be called?
```

Comment:
Je to jen zatím proof-of-concept implementace, takže některé věci chybí. Kdyžtak je klidně implementuj/přeimplementuj, nebo se se mnou domluv abych to udělal já.